airflow/docker.go

```diff
@@ -69,7 +72,12 @@ var (
 	inspectContainer = inspect.Inspect
 	initSettings     = settings.ConfigSettings
 
-	openURL = browser.OpenURL
+	openURL    = browser.OpenURL
+	timeoutNum = 60
+	tickNum    = 500
+
+	cmdPsExec   = cmdExec
+	cmdOpenExec = cmdExec
```

Comment:
why do we need this? cmdExec is already a variable in itself, so we can directly update that in our unit test. Not defining this, will reduce one extra hop for users while reading the code.

Reply:
cmdExec is ran three times during the function startDocker. Each time it needs a different output not sure how to accomplish this within the unit test. All instances seem cmdExec take on the the last definition

Reply:
in that case, the unit test can be modified as:
```Golang
t.Run("start docker success", func(t *testing.T) {
		cmdExec = func(cmd string, stdout, stderr io.Writer, args ...string) error {
                         switch cmd {
                         case "open":
                             return nil
                         case "ps":
                             return nil
                         default:
                             return errExecMock
                         }
		}

		err := startDocker()
		assert.NoError(t, err)
	})

	t.Run("start docker fail", func(t *testing.T) {
		timeoutNum = 5

		cmdExec = func(cmd string, stdout, stderr io.Writer, args ...string) error {
                         switch cmd {
                         case "open":
                             return nil
                         case "ps":
                             return errExecMock
                         default:
                             return errExecMock
                         }
		}
		err := startDocker()
		assert.Contains(t, err.Error(), "timed out waiting for docker")
	})
```

Reply:
Thanks!